templates/web/components/team_nav.html

```
@@ -154,6 +154,17 @@
             {% translate "Datasets" %}
           </a>
         </li>
+        {% flag "flag_assessments_concordance" %}
+          {% if perms.human_annotations.view_annotationqueue %}
+            <li>
+              <a href="{% url 'assessments:concordance' request.team.slug %}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)